test/Event/EventQueueItemTest.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace ComquerTest\Event;
+
+use Comquer\Event\EventQueueItem;
+use ComquerTest\Fixture\Event\ItemAdded;
+use ComquerTest\Fixture\Event\UpdateShoppingListProjection;
+use PHPUnit\Framework\TestCase;
+
+class EventQueueItemTest extends TestCase
+{
+    /** @test */
+    function getters()
```

Comment:
Is  it particularly valuable to test these getters? In general I have found that testing getters makes classes less maintainable without providing much value